src/WebJobs.Script/Utility.cs

```diff
@@ -879,6 +881,31 @@ public static bool IsResourceAzureBlobWithoutSas(Uri resourceUri)
             return true;
         }
 
+        public static string GetSasTokenExpirationDate(Uri resourceUri)
+        {
+            // Screen out URLs with an SAS token
+            var queryParams = HttpUtility.ParseQueryString(resourceUri.Query.ToLower());
+            if (queryParams != null && !string.IsNullOrEmpty(queryParams[SasTokenExpirationDate]))
```

Comment:
I think you can remove the first null check.

`if (!string.IsNullOrEmpty(queryParams[SasTokenExpirationDate]))`